config/initializers/_load_config.rb

```diff
@@ -96,6 +91,6 @@
   default.reverse_merge!(options_from_config)
 end
 
-if Rails.env.production?
-  Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
-end
+Rails.application.config.consider_all_requests_local = Errbit::Config.display_internal_errors
```

Comment:
This idea grew out of a strict interpretation of of 12factor.net:
> Another aspect of config management is grouping. Sometimes apps batch config into named groups (often called “environments”) named after specific deploys, such as the development, test, and production environments in Rails. This method does not scale cleanly: as more deploys of the app are created, new environment names are necessary, such as staging or qa. As the project grows further, developers may add their own special environments like joes-staging, resulting in a combinatorial explosion of config which makes managing deploys of the app very brittle.

I was actually toying with the idea of removing all those named environment files also because they're basically meaningless. If everything is configured through environment variables, there's not much need for grouping config. We may as well just have one environment.rb file that includes all the truly hard coded stuff and reference Errbit::Config for everything else.

I might just do this to see what it looks like in this PR. I can always take it out if it sucks.